pkg/build/api/types.go

```diff
@@ -63,6 +63,23 @@ const (
 	// forces the build to be processed by the build controller queue without waiting
 	// for a resync.
 	BuildAcceptedAnnotation = "build.openshift.io/accepted"
+
+	// BuildStartedEvent is the reason associated with the event registered when a build is started (pod is created).
+	BuildStartedEventReason = "BuildStarted"
+	// BuildStartedEvent is the message associated with the event registered when a build is started (pod is created).
+	BuildStartedEventMessage = "Pod has been created to run build %s/%s"
```

Comment:
Could the event ever apply to a pipeline build? If so, maybe a more generic message would be better.